build.sbt

```diff
@@ -327,6 +327,16 @@ lazy val zincScripted = (project in internalPath / "zinc-scripted").
   ).
   configure(addSbtUtilScripted)
 
+lazy val crossTestBridges = {
+  Command.args("crossTestBridges", "<version>") { (state, args) =>
+    require(args.nonEmpty, "Missing Scala version argument.")
+    compilerBridgeScalaVersions.flatMap { (bridgeVersion: String) =>
+      s"wow $bridgeVersion" ::
+        s"${compilerBridge.id}/test" :: Nil
```

Comment:
It went from 9 plzes to 3 wows https://github.com/sbt/zinc/pull/211/commits/d5d59b6f4c563a5f9aa5d095295b93b912b90cf3

Is it something intrinsic about plz? At the time I just thought it was just the number of them.

Anyways, not a blocker.